browsers/chrome/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM krallin/ubuntu-tini:xenial
+FROM node:16.16.0 as base
```

Comment:
Nice work, the other one was ancient

Reply:
4 years, and was not updating it's dependancies.